### PR TITLE
Refactor VBO definitions #209

### PR DESCRIPTION
**Pull request by katiermullen**
Addresses #209

### CODE REVIEW

**Comment by katiermullen** [comment]
@sabrinatoro please review at your convenience and do not merge. Thank you!

**Comment by katiermullen** [comment]
@sabrinatoro please review updates:

- Definitions of vertebrate breed and high level classifications. 
- Definitions of mixed breed dog and mixed breed cat

**Review by sabrinatoro — DISMISSED** [review]
I approve this PR. This can be merged after others have reviewed it.

File: src/ontology/components/relation.owl
Comment (matentzn) [inline src/ontology/components/relation.owl:93]: A property name should always to fully self contained, even if the sentences constructed by a person sounds odd. This usually means mentioning the domain:

```suggestion
        <rdfs:label xml:lang="en">breed reported in geographic location</rdfs:label>
```

Also feel free to make "breed" the domain of this relation, but that's not imperative here.

File: src/ontology/components/relation.owl
Comment (matentzn) [inline src/ontology/components/relation.owl:92]:
```suggestion
        <rdfs:comment>Officially nominated FAO National Coordinators report breeds existing in their country (these breeds are reported in the DAD-IS breed list). 'of breed reported in' refers to the region where the breed was reported. For example: VBO:0007427 represents the 'Alatau' breed reported to be found in Kyrgyzstan by the FAO coordinators.</rdfs:comment>
```

File: src/ontology/components/relation.owl
Comment (matentzn) [inline src/ontology/components/relation.owl:86]: Are we totally positive that `VBO:0300020` should be an Object property? Remember that it gets _always_ inherited down the hierarchy.

File: src/ontology/components/highlevelclass.tsv
Comment (matentzn) [inline src/ontology/components/highlevelclass.tsv:5]: Remember that human readable definititions (as opposed to descriptions!) should be necessary and sufficient.

```
A vertebrate breed that is a Bubalus bubalis.
``` 

Makes limited sense to my eyes.

1. Either this breed _corresponds exactly to_ `Bubalus bubalis`, and then its not needed (you just use the NCBITaxon id?) or
2. There is more than one breed that  "is a Bubalus bubalis" and then the definition is insufficient, i.e. the differentiae is missing. 

If you tell me which it is I will help you with the right solution.

File: src/ontology/components/dogbreeds.owl
Comment (matentzn) [inline src/ontology/components/dogbreeds.owl:56588]: I guess ok

File: src/ontology/components/catbreeds.owl
Comment (matentzn) [inline src/ontology/components/catbreeds.owl:1]: Too busy to review this file, can you point out a specific line that needs looking at?

**Review by matentzn — COMMENTED** [review]
Do _NOT_ apply my change suggestions, I made them only to explain what changes are needed (they are made to the wrong files!!!)

File: src/ontology/components/catbreeds.owl
Comment (katiermullen) [inline src/ontology/components/catbreeds.owl:16136]: @matentzn does this look ok?

File: src/ontology/components/relation.owl
Comment (katiermullen) [reply src/ontology/components/relation.owl:86]: I cannot think of an instance when this property should not be inherited down the hierarchy, but I better check with @sabrinatoro.

File: src/ontology/components/highlevelclass.tsv
Comment (katiermullen) [reply src/ontology/components/highlevelclass.tsv:5]: Option2: there are 4 different breeds of Bubalus bubalis (American bison).

Screenshot 2024-11-27 at 2 15 32 PM

Yes, please help me find the right solution :)

Comment (matentzn) [reply src/ontology/components/highlevelclass.tsv:5]: Here is the key question:

What is the difference between "American bison breed" and https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=9901?

File: src/ontology/components/catbreeds.owl
Comment (matentzn) [reply src/ontology/components/catbreeds.owl:16136]: If `species`  refers to the "genus" of cats, then this definition sounds a bit circular (and it has an illegal space in the end by the way)!

ChatGPT suggests something like this:

```
A cat whose ancestry includes a combination of different breeds, without a consistent pedigree linking it to a specific breed standard.
```

But even `A cat whose ancestry includes a combination of different breeds` would make a bit more sense to me? (same for dogs)

File: src/ontology/components/highlevelclass.tsv
Comment (katiermullen) [reply src/ontology/components/highlevelclass.tsv:5]: I would have to say nothing. I think the rationale for adding the grouping classes, e.g. American bison breed is on potentially on shaky ground. These grouping terms (e.g. chicken breed, duck breed, cattle breed, dog breed, etc.) were added to make it easier to search for individual breeds in the hierarchy. 

Maybe in the case of cattle breed where the grouping class is at the level of Bos and therefore includes all members of Bos https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi, there is a difference between, for example bos taurus and "cattle breed," but this is not the case for grouping classes that are defined at the species or subspecies level.  

What do you recommend?

File: src/ontology/components/catbreeds.owl
Comment (katiermullen) [reply src/ontology/components/catbreeds.owl:16136]: On the topic of cats, see also this user request: https://github.com/monarch-initiative/vertebrate-breed-ontology/issues/211

for replacing the term "mixed breed cat" with "random bred cat." I don't think that "random bred cat" can replace "mixed breed cat" because they have different definitions, but I think that we can add this new term to VBO.

File: src/ontology/components/highlevelclass.tsv
Comment (matentzn) [reply src/ontology/components/highlevelclass.tsv:5]: If they are the same you will have to consult with @sabrinatoro and consider swapping in NCBITaxon ids.. How much effort would that be?

File: src/ontology/components/relation.owl
Comment (sabrinatoro) [reply src/ontology/components/relation.owl:86]: I agree with @katiermullen, any children of a term with this relation should always inherit this relation. 
This relation is to be used to define a term. So all children should also have it (note that I doubt that we would have a lot of children for these terms)

File: src/ontology/components/highlevelclass.tsv
Comment (sabrinatoro) [reply src/ontology/components/highlevelclass.tsv:5]: These 2 taxons are different in NCBITaxon though:
"Bulbus" and "Bison" are 2 distinct groups of [Bovinae](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Undef&id=27592&lvl=3&lin=f&keep=1&srchmode=1&unlock)
Are we saying that the breeds should be children of both?

Comment (matentzn) [reply src/ontology/components/highlevelclass.tsv:5]: If they are different then the answer to

> What is the difference between "American bison breed" and https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=9901?

Is not "nothing"; @katiermullen do I see this correctly that Sabrina believes they are indeed different?

File: src/ontology/components/relation.owl
Comment (matentzn) [reply src/ontology/components/relation.owl:86]: Great! Closing this thread.

**Comment by katiermullen** [comment]
@sabrinatoro please review:

- 'vertebrate breed' definition
- high level classification definitions (e.g. 'dog breed')
- 'breed reported in geographic location' relation
- 'mixed breed cat' definition
- 'mixed breed dog' definition

Thank you!!!

**Comment by katiermullen** [comment]
@matentzn we have updated the definition of dog breed to "A breed of Canis lupis familiaris." Sabrina and I want to make sure that you think this is okay since we define "Vertebrate breed" and not "breed."

**Comment by matentzn** [comment]
This is correct, using "vertebrate" in this context is redundant as the taxon is already vertebrate.

**Comment by katiermullen** [comment]
@sabrinatoro ready for your review!

@cmungall we have updated the definition of "dog breed" to "A breed of Canis lupis familiaris." 

For convenience, "vertebrate breed" definition is: "A taxonomic entity representing a population of vertebrate animals that share specific characteristics (such as traits, behavior, genetics) and/or for which cultural or geographical separation has led to the general acceptance of its separate identity."

Please let us know if you have any comments, otherwise we will forge ahead!

**Review by sabrinatoro — APPROVED** [review]
@katiermullen I approve this PR. Great job! 
I think it is ready to merge, but in case there was one last thing, I will let you merge it yourself.
Truly great job!

**Pull request merged**
